src/spikeinterface/extractors/iblextractors.py

```diff
@@ -0,0 +1,355 @@
+        self.set_property("adc", adc)
+        self.set_property("index_on_probe", index_on_probe)
+        if not all(good_channel):
+            self.set_property("good_channel", good_channel)
+
+        # init recording segment
+        recording_segment = IblRecordingSegment(file_streamer=self._file_streamer, load_sync_channel=load_sync_channel)
+        self.add_recording_segment(recording_segment)
+
+        self._kwargs = {
+            "eid": eid,
+            "pid": pid,
+            "stream_name": stream_name,
+            "load_sync_channel": load_sync_channel,
+            "cache_folder": cache_folder,
+            "remove_cached": remove_cached,
+            "stream": stream,
+        }
+
+
+class IblRecordingSegment(BaseRecordingSegment):
+    def __init__(self, file_streamer, load_sync_channel: bool = False):
+        BaseRecordingSegment.__init__(self, sampling_frequency=file_streamer.fs)
+        self._file_streamer = file_streamer
+        self._load_sync_channel = load_sync_channel
+
+    def get_num_samples(self):
+        return self._file_streamer.ns
+
+    def get_traces(self, start_frame: int, end_frame: int, channel_indices):
+        if start_frame is None:
+            start_frame = 0
+        if end_frame is None:
+            end_frame = self.get_num_samples()
+        if channel_indices is None:
+            channel_indices = slice(None)
+        traces = self._file_streamer.read(nsel=slice(start_frame, end_frame), volts=False)
+        if not self._load_sync_channel:
+            traces = traces[:, :-1]
+
+        return traces[:, channel_indices]
+
+
+class IblSortingExtractor(BaseSorting):
+    """Load IBL data as a sorting extractor.
+
+    Parameters
+    ----------
+    pid: str
+        Probe insertion UUID in Alyx. To retrieve the PID from a session (or eid), use the following code:
+
+        >>> from one.api import ONE
+        >>> one = ONE(base_url="https://openalyx.internationalbrainlab.org", password="international", silent=True)
+        >>> pids, _ = one.eid2pid("session_eid")
+        >>> pid = pids[0]
+    good_clusters_only: bool, default: False
+        If True, only load the good clusters
+    load_unit_properties: bool, default: True
+        If True, load the unit properties from the IBL database
+    one: One | dict, default: None
+        Instance of ONE.api or dict to use for data loading.
+        For multi-processing applications, this can also be a dictionary of ONE.api arguments
+        For example: one={} or one=dict(base_url='https://alyx.internationalbrainlab.org', mode='remote')
+
+    Returns
+    -------
+    extractor : IBLSortingExtractor
+        The loaded data.
+    """
+
+    extractor_name = "IBLSorting"
+    name = "ibl"
+    installation_mesg = "IBL extractors require ibllib as a dependency." " To install, run: \n\n pip install ibllib\n\n"
+
+    def __init__(self, pid, good_clusters_only=False, load_unit_properties=True, one=None):
```

Comment:
@h-mayorquin added the `good_clusters_only` to only get units passing IBL QC. Also, with `load_unit_properties=True` the brain location is loaded as the `acronym` property. 

We should use `good_clusters_only=True` to build a first round of _potentially_ high-quality templates

Reply:
One question here: Could we store the brain_location as a property named brain_location instead of "acronym"?

Reply:
yup, pushed a patch to use `brain_area`